edx_oauth_client/middleware.py

```diff
@@ -69,7 +69,15 @@ def process_request(self, request):
         """
         Redirect to PLP for pages that have duplicated functionality on PLP.
         """
-        PROVIDER_URL = settings.get('CUSTOM_OAUTH_PARAMS', {}).get("PROVIDER_URL", "")
+        CUSTOM_OAUTH_PARAMS = {}
+        if hasattr(settings, 'CUSTOM_OAUTH_PARAMS'):
```

Comment:
I would rewrite lines 72-74 and it will see like:
`
CUSTOM_OAUTH_PARAMS = settings.CUSTOM_OAUTH_PARAMS if hasattr(settings, 'CUSTOM_OAUTH_PARAMS') else {}
`

Reply:
Fixed